src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Ball {
+    private Color color;
+    private int number;
```

Comment:
add getters and setters, or you won't be able to access these fields after their initialization